courses/management/commands/manage_program_certificates.py

```diff
@@ -0,0 +1,144 @@
+"""
+Management command to revoke, un revoke or create a certificate for a program for the given User.
+
+Check the usages of this command below:
+
+**Certificate Creation**
+
+1. Generate Pogram certificate for a user
+./manage.py manage_program_certificates —-create -—program=<program_readable_id> -—user=<username or email>
+
+**Revoke/Un-revoke Certificates**
+
+2. Revoke a certificate for a user
+./mange.py manage_program_certificates -—revoke -—user=<username or email> -—program=<program_readable_id>
+
+3. Un-Revoke a certificate for a user
+./mange.py manage_program_certificates -—unrevoke —-program=<program_readable_id> -—user=<username or email>
+
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from users.api import fetch_user
+from courses.api import manage_program_certificate_access, generate_program_certificate
+from courses.models import Program
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Invoke with:
+
+        python manage.py manage_program_certificates
+    """
+
+    help = (
+        "Revoke, un revoke or create a certificate for a program for the given User "
+        "or Users when no user is provided"
```

Comment:
It's hard for me to imagine a situation where we would want to scan all the learners in a program and create certificates for them all. 

This command should be limited to one user, and it should create the program certificate no matter if they have passed the required courses.

Reply:
>This command should be limited to one user, and it should create the program certificate no matter if they have passed the required courses.

Thanks for the clarification, so this is going to be for one user. 

@pdpinch as per what you mentioned, I think we won't need the `--force` flag at all if we don't want to check the passed status at all, Otherwise, I'm mentioning my perceptions below:

- In normal conditions, the command will check the courses' passed status and generate the letter if the required courses are passed.
- If we don't want to check the passed status, we can use `--f` for that as you mentioned.